content/docs/operating/integrations.md

```diff
@@ -49,6 +49,7 @@ data volumes.
   * [SignalFx](https://github.com/signalfx/metricproxy#prometheus): write
   * [Splunk](https://github.com/lukemonahan/splunk_modinput_prometheus#prometheus-remote-write): write
   * [TiKV](https://github.com/bragfoo/TiPrometheus): read and write
+  * [Thanos](http://github.com/improbable-eng/thanos): write and read client
```

Comment:
I thought you only supported write? Can I read from Thanos via remote read?

Reply:
I explained that in my PR description (:

Reply:
Happy to move it to `write` only, depending how you define integration here, as we hook into remote read (:

Reply:
The question is whether Prometheus can read/write, not whether something uses read/write itself.

Reply:
Agreed, for now (and potentially ever) thanos is going to be wrote only.

Reply:
agree, changed.